src/content/advanced/ingress/configuration/index.md

```diff
@@ -443,11 +431,12 @@ metadata:
   namespace: NAMESPACE
 data:
   values: |
-    configmap:
-      log-format-upstream: "MY EDITED LOG FORMAT - $status $body_bytes_sent $http_referer"
+    controller:
+      config:
+        log-format-upstream: "MY EDITED LOG FORMAT - $status $body_bytes_sent $http_referer"
 ```
 
-However keep in mind that with great power comes great responsibility.
+However keep in mind: With great power comes great responsibility! 🕷️
```

Comment:
Not sure if customers know what the bug depicts at this specific location 😉

Reply:
Yeah, you're probably right. I was already thinking about linking the Wiki page (there actually is one about this quote), but gonna remove the emoji.